### PR TITLE
Add lesson14

### PR DESCRIPTION
**Pull request by haraldF**

### CODE REVIEW

File: lesson14.md [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+```js
+let myDivElement = document.getElementById("myDiv");s
+let myInputElement = document.getElementById("myInput");
+let myButtonElement = document.getElementById("myButton");
+```
+
+---
+
+```js
+let myDivElement = document.getElementById("myDiv");s
```
Comment (carlotrimarchi) [inline lesson14.md]: there's an "s" too much

File: lesson14.md [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+1. Create a button, and call a function in its `onclick` handler
+1. Change some of the attributes of your HTML Elements
+
+---
+
+### Let's write a shop!
+
+1. Create some HTML, with two (or more) products you would like to sell in your shop. Every product should have a small description, a price and a `<input type="number">` element for selecting the amount.
+1. Assign a function to each input element's `oninput` property.
+1. Use the `valueAsNumber` property of your input element to get the value as `number` (not string).
```
Comment (carlotrimarchi) [inline lesson14.md]: I think that this step is not really clear

File: lesson14.md (hunk 1 above)
Comment (haraldF) [reply lesson14.md]: Haha yeah, my command key gets stuck sometimes. So now you know I tried to save the file at that point ;)

File: lesson14.md (hunk 2 above)
Comment (haraldF) [reply lesson14.md]: ok, I've added a bit more description

Comment (carlotrimarchi) [reply lesson14.md]: actually I was confused by `valueAsNumber`. So, just googled it, I had no idea it was an actual property 😅

**Pull request merged**